client/state/preferences/README.md

```diff
@@ -45,5 +45,6 @@ To add a new preference key, you only need to add a key and defaut value in `DEF
 ### Actions
 
 - **fetchPreferences()** - populate the tree
-- **setPreference( key, value )** - set preference and persist all preferences in the endpoint
+- **savePreference( key, value )** - set preference and persist all preferences in the endpoint
```

Comment:
> Would we ever want to sometimes store a preference for a session, and later store it forever?

Yes, that's precisely what happens here. We can disable showing this tooltip either forever, or only for current session: https://cloud.githubusercontent.com/assets/3775068/16149124/c8f567de-348e-11e6-9bbc-8290e2fce512.png